### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by AkshayGupta8**
Created Footer with updated icon sizing.

Override LinkComponents props with `font-size` attribute updated.

Added LinkComponent prop with default argument Link in ContactLinks.jsx

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <FooterBody>
+            <ConnectWithUsFooter>
+                CONNECT WITH US
+            </ConnectWithUsFooter>
+            <ContactLinks LinkComponent={CustomLink} />
+        </FooterBody>
+    </FooterOuter>
+);
+
+export default Footer;
```
Comment (KindaOK) [inline src/components/Footer.jsx:95]: This component isn't actually used anywhere. It should be at the bottom of all pages other than home.

File: src/components/Contact/ContactLinks.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const LinksContainer = styled.div`
     `}
 `;
 
-const Link = styled.a`
+export const Link = styled.a`
```
Comment (KindaOK) [inline src/components/Contact/ContactLinks.jsx:19]: Don't export components from here for use in the footer. If you see that this component can be deduplicated, then extract it to `ContentStyles.js` and import it into these two components.

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    `}
+`;
+
+const ConnectWithUsFooter = styled.h1`
+    color: white;
+    text-align: center;
+    font-weight: 700;
+`;
+
+const CustomLink = styled(Link)`
```
Comment (KindaOK) [inline src/components/Footer.jsx]: I don't think this is worth using inheritance for. While this is something we should improve, this is not the best place to start.

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+// import { NavLink } from "react-router-dom";
+import devices from "../utility/MediaQueries.js";
+// import logoUrl from "assets/logo-orange-cropped.png";
+// import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (KindaOK) [inline src/components/Footer.jsx]: Remove comments from imports

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+// import { NavLink } from "react-router-dom";
+import devices from "../utility/MediaQueries.js";
```
Comment (KindaOK) [inline src/components/Footer.jsx]: Use absolute imports instead of relative imports

File: src/components/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+// import logoUrl from "assets/logo-orange-cropped.png";
+// import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+// import { SOCIAL_MEDIA_LINKS } from "utility/constants.js";
+import ContactLinks, { Link } from "./Contact/ContactLinks";
```
Comment (KindaOK) [inline src/components/Footer.jsx]: Include file extensions in imports

File: src/components/Footer.jsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+`;
+
+
+const Footer = () => (
+    <FooterOuter>
+        <FooterBody>
+            <ConnectWithUsFooter>
+                CONNECT WITH US
+            </ConnectWithUsFooter>
+            <ContactLinks LinkComponent={CustomLink} />
```
Comment (KindaOK) [inline src/components/Footer.jsx]: This is probably a good idea given how many places we have this. However, this execution is not great. Refactor out the fragments into smaller components in a high-level file.

File: src/components/Footer.jsx [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    font-weight: 700;
+`;
+
+const CustomLink = styled(Link)`
+  font-size: 3rem;
+`;
+
+
+const Footer = () => (
+    <FooterOuter>
```
Comment (KindaOK) [inline src/components/Footer.jsx:53]: Reduce the amount of wrappers if possible. These seem to be redundant.

File: src/components/Footer.jsx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    width: min(60%, 600px);
+    ${devices.tablet`
+        width: min(80%, 600px);
+    `}
+    ${devices.small`
+        width: min(90%, 600px);
+    `}
+`;
+
+const ConnectWithUsFooter = styled.h1`
```
Comment (KindaOK) [inline src/components/Footer.jsx]: There should be no more than one `<h1>` per page. I don't think this should even be a heading given that this is generic content at the bottom of each page.

File: src/components/Footer.jsx [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+const CustomLink = styled(Link)`
+  font-size: 3rem;
+`;
+
+
+const Footer = () => (
+    <FooterOuter>
+        <FooterBody>
+            <ConnectWithUsFooter>
+                CONNECT WITH US
```
Comment (KindaOK) [inline src/components/Footer.jsx]: The actual design seems fine. The text and element itself is fairly large though.

**Review by KindaOK — CHANGES_REQUESTED** [review]
Needs several changes. Read [this article](https://cbea.ms/git-commit/) for writing better commit messages as well.

File: src/components/Footer.jsx [hunk 11]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+// import { NavLink } from "react-router-dom";
+import devices from "../utility/MediaQueries.js";
+// import logoUrl from "assets/logo-orange-cropped.png";
+// import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+// import { SOCIAL_MEDIA_LINKS } from "utility/constants.js";
+import ContactLinks, { Link } from "./Contact/ContactLinks";
+
+const FooterOuter = styled.div`
```
Comment (KindaOK) [inline src/components/Footer.jsx]: The outermost wrapper should be a `<footer>`

File: src/components/Footer.jsx [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: rgba(238, 95, 48, 1);
```
Comment (KindaOK) [inline src/components/Footer.jsx]: Color should be hacker orange (see header)

**Review by KindaOK — CHANGES_REQUESTED** [review]

File: src/components/Footer.jsx (hunk 6 above)
Comment (kgurushankar) [inline src/components/Footer.jsx]:
```suggestion
import devices from "../utility/MediaQueries.js";
import ContactLinks, { Link } from "./Contact/ContactLinks.jsx";
```

**Review by KindaOK — APPROVED** [review]

**Pull request merged**